### ISSUE TITLE
Optimize Open Graph on localized pages

### ISSUE THREAD

**Issue by gideonthomas** [mozilla/foundation.mozilla.org#3474]
Update og:locale, or set og:locale:alternate on localized pages.

**Problem**
We’re currently using this everywhere: `<meta property='og:locale' content='en_US'>`

**Comment by Pomax** [comment]
This was added in a PR I was working on, I'll see which one it is and close-cross-link.

**Comment by Pomax** [comment]
Addressed in https://github.com/mozilla/foundation.mozilla.org/pull/3485/files#diff-466035fbdea07f4221da7ce361860856R6

**Comment by Pomax** [comment]
extracted as separate PR to unblock other work.
